sql-ballerina/Module.md

```diff
@@ -0,0 +1,302 @@
+## Overview
+
+This module provides the common interface and functionality to interact with a database. The corresponding database
+clients can be created by using specific database modules such as `MySQL` or using the Java Database Connectivity 
```

Comment:
```suggestion
clients can be created by using specific database modules such as `mysql` or using the Java Database Connectivity 
```

Reply:
Added suggested change in https://github.com/ballerina-platform/module-ballerina-sql/pull/228/commits/fa55a74e7928c9529090430cd4a5a358e2af8e8e